data/quickstarts/deploy-python-model-quickstart.yaml

```diff
@@ -8,69 +8,75 @@ spec:
   description: How to deploy a Python model using Flask and OpenShift.
   prerequisites: [You completed the "Create a Jupyter notebook" quick start.]
   introduction: |-
-    ### This quick start shows you how to deploy a Python Model.
-    Red Hat® OpenShift® Data Scientist helps you deploy a model on our Red Hat® OpenShift Dedicated environment.
+    ### This quick start shows you how to deploy a Python model.
+    Red Hat® OpenShift® Data Science makes it easy for you to deploy your data science models in a Red Hat® OpenShift® Dedicated environment.
 
-    We have configured a template repository that uses s2i to support deploying your amazing Data Scientist explorations using Flask on OpenShift.
-
-    We just know you will absolutely love this amazing environment.  This quick start will get your model deployed for sandbox development usage.
+    This quick start shows you how to take your model out of a Jupyter notebook and put it into a Flask application running on OpenShift Dedicated, for use as a development sandbox.
   tasks:
-    - title: Create a new repository from the s2i Flask github template
+    - title: Create a new GitHub repository using the s2i Flask template
       description: |-
-        ### To create a new repository from the 2si Flask template:
+        ### To create a new repository from the s2i Flask template:
         1. Navigate to `https://github.com/cfchase/s2i-flask-notebook`.
```

Comment:
This repo has been transferred to https://github.com/opendatahub-io/odh-s2i-project-simple

Reply:
Thanks! Will update today.